src/components/explorer/ExplorerHeader.tsx

```diff
@@ -0,0 +1,333 @@
+import { buttonStyles } from 'components/common/button/buttonStyles';
+import { TokensOverlap } from 'components/common/tokensOverlap';
+import { useTokens } from 'hooks/useTokens';
+import { Strategy, useGetStrategyList } from 'libs/queries';
+import {
+  PairTrade,
+  Trending,
+  useTrending,
+} from 'libs/queries/extApi/tradeCount';
+import { Link } from 'libs/routing';
+import { CSSProperties, useEffect, useRef } from 'react';
+import { toPairSlug } from 'utils/pairSearch';
+
+const useTrendingPairs = (trending?: Trending) => {
+  const { tokensMap } = useTokens();
+  if (!trending) return { isLoading: true, data: [] };
+  const pairs: Record<string, PairTrade> = {};
+  for (const trade of trending?.pairCount ?? []) {
+    pairs[trade.pairAddresses] ||= trade;
+  }
+  const list = Object.values(pairs)
+    .filter((pair) => !!pair.pairTrades_24h)
+    .sort((a, b) => b.pairTrades - a.pairTrades)
+    .splice(0, 3);
+
+  // If there are less than 3, pick the remaining best
+  if (list.length < 3) {
+    const remaining = Object.values(pairs)
+      .filter((pair) => !!pair.pairTrades_24h)
+      .sort((a, b) => b.pairTrades - a.pairTrades)
+      .splice(0, 3 - list.length);
+    list.push(...remaining);
+  }
```

Comment:
```suggestion
  // If there are less than 3, pick the remaining best
  if (list.length < 3) {
    const remaining = structuredClone(list).splice(0, 3 - list.length);
    list.push(...remaining);
  }
  }
```

Reply:
We need to sort back with only the item that has pairTrades_24h. This is why I need to filter & sort again.